<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchResultWidget</class>
 <widget class="QWidget" name="SearchResultWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>918</width>
    <height>716</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SearchResultWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="clearPushButton">
       <property name="toolTip">
        <string>Clear search results</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="infoviewer.qrc">
         <normaloff>:/InfoViewer/images/clear.png</normaloff>:/InfoViewer/images/clear.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QScrollArea" name="searchScrollArea">
     <property name="widgetResizable">
      <bool>false</bool>
     </property>
     <widget class="CustomTextViewWidget" name="searchResultViewWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>912</width>
        <height>682</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CustomTextViewWidget</class>
   <extends>QWidget</extends>
   <header>CustomTextViewWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="infoviewer.qrc"/>
 </resources>
 <connections/>
</ui>
